### PR TITLE
Demo screen

### LINKED ISSUE #10: Demo Page
# Goal

Create the demo page reachable from the landing page.

# Solution

The scrolling carousel can be implemented using [`react-native-reanimated-carousel`](https://www.npmjs.com/package/react-native-reanimated-carousel).

# Tasks
- [ ] Carousel images
  - [ ] Image 1
  - [ ] Image 2
  - [ ] Image 3

### PR DESCRIPTION
**Pull request by hope5487**
## Problem

Complete the demo screen

Closes #10 

## Description of Changes

- Changes in App.tsx (added new fonts and make "see a demo" button to navigate the demo screen)

- Complete the demo screen

image

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: screens/WelcomeScreen.tsx [hunk 1]
```diff
@@ -46,7 +46,8 @@ function WelcomeScreen({ navigation }) {
                     </View>
                 </LinearGradient>
             </Pressable>
-            <Pressable style={styles.demoContainer}>
+            <View style={styles.demoShadowContainer} />
```
Comment (charleswong739) [inline screens/WelcomeScreen.tsx:49]: There is no corresponding style for this View + `demoContainer` defines its own shadow. Was there another purpose in mind for this empty View?

File: App.tsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ import HomeScreen from './screens/HomeScreen';
 import LoginScreen from './screens/LoginScreen';
 import WelcomeScreen from './screens/WelcomeScreen';
 import SignUpScreen from './screens/SignUpScreen';
+import DemoScreen from './screens/DemoScreen';
```
Comment (charleswong739) [inline App.tsx:16]: `DemoScreen` is imported but not actually added to the login navigation stack. See these lines: https://github.com/ashenafee/STG/blob/13de0aa5d4436a12b05a528d9b1a5dcf92ac5400/App.tsx#L121-L123

File: screens/DemoScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+                <Image source={require('../assets/continue.png')} style={styles.continueImage}/>
+            </TouchableOpacity>
+        </View>
+    );
+}
+
+const styles = EStyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: '#fff',
```
Comment (charleswong739) [inline screens/DemoScreen.tsx:56]: The background colour for the app is not pure white. Rebase this branch onto `main` to make this branch compatible with current development, and consider using the [`Screen`](https://github.com/ashenafee/STG/blob/main/components/Screen.tsx) component.

**Review by charleswong739 — CHANGES_REQUESTED** [review]

File: screens/DemoScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+            </View>
+
+            <View style={styles.firstDemo}>
+                <Image source={require('../assets/demo_camera.png')} style={styles.demoCamera}/>
+                <Image source={require('../assets/demo_upload.png')} style={styles.demoUpload}/>
+            </View>
+
+            <View style={styles.secondDemo}>
+                <Image source={require('../assets/demo2.png')} style={styles.numberSecond}/>
+                <Text style={styles.descriptionText}>snap a photo of your item 📸</Text>
```
[the remark this answers is not in the document]
Comment (hope5487) [reply screens/DemoScreen.tsx]: Fixed

File: screens/DemoScreen.tsx
[the remark this answers is not in the document]
Comment (hope5487) [reply screens/DemoScreen.tsx:1]: Fixed

File: screens/DemoScreen.tsx (hunk 3 above)
Comment (hope5487) [reply screens/DemoScreen.tsx]: Fixed

**Review by charleswong739 — APPROVED** [review]

**Pull request merged**